src/Codec/StringCodec.php

```diff
@@ -29,7 +29,7 @@ class StringCodec implements CodecInterface
     /**
      * @var UuidBuilderInterface
      */
-    private $builder;
+    protected $builder;
```

Comment:
It's a bit of nitpicking, but I find it better to leave the property private and add a protected getter to prevent accidental overwrites.